kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -687,58 +690,106 @@ define([
          */
         function renderLayout() {
             const uniqueID = html.genId(),
-                events = Events.make(),
-                content = div(
+                events = Events.make();
+            let content = div(
```

Comment:
git doesn't understand what has happened here. I've changed `content` from a `const` to a `let`; the DOM structure assigned to `content` has not changed.